spire/templates/dovetail-redis/cluster.yml

```diff
@@ -16,7 +16,7 @@ Parameters:
   VpcPrivateSubnet1Id: { Type: AWS::EC2::Subnet::Id }
   VpcPrivateSubnet2Id: { Type: AWS::EC2::Subnet::Id }
   VpcPrivateSubnet3Id: { Type: AWS::EC2::Subnet::Id }
-  SharedRedisClusterSecurityGroupId: { Type: AWS::EC2::SecurityGroup::Id }
+  DovetailRedisClusterSecurityGroupId: { Type: AWS::EC2::SecurityGroup::Id }
```

Comment:
Renaming the param won't recreate the cluster, right?  If this seems dangerous, I can leave it as-is.  But was trying to standardize names in the root stack.